### PR TITLE
Update/#222 product page update

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
Create and complete all responsives of Products Page (The link to the page is in the Navbar or use url ('./products')).

In the landing(home) page, create anchor links to specific section of the Products page. These are the links in the Scalable and Flexible section of the landing page. I created a ticket to come back to the offset. The mobile and tablet view to the Products Page via these anchor link are slightly off at the top. I tried my best without create complicated codes to resolve this in all views but the mobile and tablet views.

### CODE REVIEW

File: src/components/HomeProductLink/index.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import { LinkHelper as Link, Icon, P1 } from 'components';
+import { AnchorLink } from 'gatsby-plugin-anchor-links';
 import './styles.scss';
 
 export default function HomeProductLink({ icon, title, link, text }) {
   return (
-    <Link className="ProductLink" to={link}>
+    <AnchorLink className="ProductLink" to={link}>
```
Comment (ciaranschutte) [inline src/components/HomeProductLink/index.js:8]: Why using `AnchorLink` directly? The previous `Link` component also uses a gatsby link: https://github.com/overture-stack/website/blob/develop/src/components/LinkHelper/index.js

Comment (daniel-cy-lu) [reply src/components/HomeProductLink/index.js:8]: Let me investigate. I just read the code in HelperLink and want to test it out.

Comment (ciaranschutte) [reply src/components/HomeProductLink/index.js:8]: hold off on this please. we'll just target MVP things. see slack.

Comment (daniel-cy-lu) [reply src/components/HomeProductLink/index.js:8]: Ok! will do that!

**Review by ciaranschutte — APPROVED** [review]

**Pull request merged**